client/package.json

```diff
@@ -25,6 +25,8 @@
     "polished": "^3.5.1",
     "prettier": "^2.0.2",
     "react": "^16.13.1",
+    "react-alert": "^7.0.0",
```

Comment:
Also, @markthien not that I want to annoy you about changing everything but why did you add an extra component and theme to show the alerts?
There is also an alert component in Ant Design https://ant.design/components/alert/ 
This will probably also be good enough?
It's just that we should try to use less dependencies, so try to use Ant Design / Ant Design Mobile as much as possible... sorry if that wasn't clear enough, maybe we need to update that in the documentation.